src/neuroconv/datainterfaces/behavior/video/externalvideointerface.py

```diff
@@ -0,0 +1,330 @@
+from copy import deepcopy
+from typing import Literal, Optional
+
+import numpy as np
+from pydantic import FilePath, validate_call
+from pynwb import NWBFile
+from pynwb.image import ImageSeries
+
+from .video_utils import VideoCaptureContext
+from ....basedatainterface import BaseDataInterface
+from ....tools import get_package
+from ....tools.nwb_helpers import get_module
+from ....utils import dict_deep_update, get_base_schema, get_schema_from_hdmf_class
+
+
+class ExternalVideoInterface(BaseDataInterface):
+    """Data interface for writing videos as ImageSeries."""
+
+    display_name = "Video"
+    keywords = ("movie", "natural behavior", "tracking")
+    associated_suffixes = (".mp4", ".avi", ".wmv", ".mov", ".flx", ".mkv")
+    # Other suffixes, while they can be opened by OpenCV, are not supported by DANDI so should probably not list here
+    info = "Interface for handling standard video file formats."
+
+    @validate_call
+    def __init__(
+        self,
+        file_paths: list[FilePath],
+        verbose: bool = False,
+        *,
+        video_name: str = "Video1",
```

Comment:
I think this name is a bit confusing when passing the file paths of multiple videos.

I don't have a good idea now on how to improve it but I am flagging this for myself and to hear your thoughts.

Reply:
Can we call it just `ExternalVideo` (without the number 1)?